psignifit/tests/test_sigmoids.py

```diff
@@ -42,26 +42,47 @@ def test_sigmoid_by_name(sigmoid_name):
 
 @pytest.mark.parametrize('sigmoid_name', sigmoids.ALL_SIGMOID_NAMES)
 def test_sigmoid_sanity_check(sigmoid_name):
```

Comment:
very cool! I think this is enough basic testing also for future sigmoids. Maybe we could expose this to the user so that they can test "home-made" sigmoids? Or would this be too much of a mess?

Reply:
Great idea! Did this